src/core/entity/value-object/__test__/name.value-object.test.ts

```diff
@@ -0,0 +1,13 @@
+describe('TESTANDO SE O NOME É UMA STRING VAZIA', () => {
+    it('Step 1', () => {
+        const name = ' ';
+        expect(name).not.toBe('DEV');
+    });
+});
+
+describe('TESTANDO SE O NOME POSSUI PELO MENOS 3 LETRAS', () => {
```

Comment:
Um bloco **describe** pode ter vários blocos **it**

assim sendo, podes remover esse bloco **describe** e colocar o it no bloco anterior